src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
```

Comment:
let's remove static

Reply:
this should not be the main goal when writing code)
you need to follow certain architectural principles

Reply:
removed.  =)